src/node_report.cc

```diff
@@ -732,13 +732,13 @@ static void PrintSystemInformation(JSONWriter* writer) {
     int id;
   } rlimit_strings[] = {
     {"core_file_size_blocks", RLIMIT_CORE},
-    {"data_seg_size_kbytes", RLIMIT_DATA},
+    {"data_seg_size_bytes", RLIMIT_DATA},
```

Comment:
In order to keep compatibility of the presense of these keys, would you mind adding both `data_seg_size_kbytes` and `data_seg_size_bytes` (and the others), with correct values?

Reply:
After the discussion below, I still removed the `*_kbytes` keys and added `*_bytes`. However, I bumped the report version from 4 to 5 and recorded the changes in the `Version history` section.